plugins/out_azure/azure.c

```diff
@@ -95,9 +98,27 @@ static int azure_format(const void *in_buf, size_t in_bytes,
         /* Append the time key */
         msgpack_pack_str(&mp_pck, flb_sds_len(ctx->time_key));
         msgpack_pack_str_body(&mp_pck,
-                              ctx->time_key,
-                              flb_sds_len(ctx->time_key));
-        msgpack_pack_double(&mp_pck, t);
+                            ctx->time_key,
+                            flb_sds_len(ctx->time_key));
+
+        if (ctx->time_generated == FLB_TRUE) {
+            /* Append the time value as ISO 8601 */
+            gmtime_r(&tm.tm.tv_sec, &tms);
```

Comment:
Is `gmtime_r` supported on Windows system ?  Since we have to support to build fluent-bit on Windows.

https://linux.die.net/man/3/gmtime_r
```
Thread-safe versions asctime_r(), ctime_r(), gmtime_r() and localtime_r() are specified by SUSv2, and available since libc 5.2.5.
```
I think it is an unix api.

Hmm, Appvayor is succeeded, so Windows supports it?

Reply:
fluent-bit has a compatibility layer: https://github.com/fluent/fluent-bit/blob/33f80ddc85443316fabd9877f4c0024956fe1e36/include/fluent-bit/flb_compat.h#L73-L79

Reply:
Thank you for pointing it.